elastic/kibana
# [Security Solutions] Add context to risk score visualisations attached to cases

## Summary

* Add description to the Lens attachment when adding the risk score visualisation to a case.
* Set the visualisation width inside cases attachment 
![Screenshot 2024-01-25 at 12 03 32](https://github.com/elastic/kibana/assets/1490444/fa3a4181-d0b8-456f-9ec0-6e74e8e22a98)



* Update the time range to use static dates instead of relative dates to make the attached visualisation a snapshot of the period.
![Screenshot 2024-01-25 at 12 04 28](https://github.com/elastic/kibana/assets/1490444/de24986c-b3be-41e5-b55d-127e4532a52a)



### How to test it?


* Enable the experimental flags `newUserDetailsFlyout` and `newHostDetailsFlyout`
* Load events into security solutions
* Create alerts for those events
* Enable the risk engine 
* Open entity analytics dashboard to check if it has generated data (if data doesn't immediately show up restart the engine to force it to run)
* Go to the alerts page and click on a `host.name` or `user.name` value
* On the opened entity details flyout interact with the lens visualisation to add it to a case
* Open the case that you attached the visualisation to and check the comment.

![Screenshot 2024-01-25 at 13 45 08](https://github.com/elastic/kibana/assets/1490444/87c0fa82-d61f-4534-a9ee-dcfd74ef826d)


### Checklist

Delete any items that are not applicable to this PR.
